packages/block-library/src/page-list/edit.js

```diff
@@ -183,63 +183,66 @@ export default function PageListEdit( {
 		style: { ...context.style?.color },
 	} );
 
-	const getBlockList = ( parentId = parentPageID ) => {
-		const childPages = pagesByParentId.get( parentId );
+	const pagesTree = useMemo(
+		function makePagesTree( parentId = 0, level = 0 ) {
```

Comment:
That's not from this PR, but it's a bit weird to see the function declaration inside `useMemo`. Can't we declare the function outside the components?

Reply:
As long as it's valid JS, I don't see a problem :)

Reply:
Lets fix this in a followup.